src/main/java/gift/category/domain/Category.java

```diff
@@ -0,0 +1,54 @@
+package gift.category.domain;
+
+import gift.product.domain.Product;
+import gift.wish.domain.WishlistItem;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotNull;
+import java.util.List;
+
+@Entity
+@Table(name="categories")
+public class Category {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    Long id;
+    @NotNull
+    String name;
+
+    @OneToMany(mappedBy = "category")
+    private List<Product> productList;
+
+    public Category() {
```

Comment:
여기서도 마찬가지로 protected로 사용하면 좋을 것 같습니다ㅎㅎ
```suggestion
    protected Category() {
```